--- functions.orig	2022-10-17 20:28:32.670048320 +0300
+++ functions	2022-10-17 20:43:56.900641966 +0300
@@ -1158,7 +1158,12 @@
 	    *freebsd*)
 		umount_on_exit /dev
 		umount_on_exit /proc
-		umount "$TARGET/proc" 2>/dev/null || true
+		if [ -L "$TARGET/proc" ];then
+			rm -f $TARGET/proc
+			mkdir $TARGET/proc
+		else
+			umount "$TARGET/proc" 2>/dev/null || true
+		fi		
 		if [ "$HOST_OS" = kfreebsd ]; then
 			in_target mount -t linprocfs proc /proc
 		else
@@ -1176,7 +1181,12 @@
 		umount_on_exit /dev/shm
 		umount_on_exit /proc
 		umount_on_exit /proc/bus/usb
-		umount "$TARGET/proc" 2>/dev/null || true
+		if [ -L "$TARGET/proc" ];then
+			rm -f $TARGET/proc
+			mkdir $TARGET/proc
+		else
+			umount "$TARGET/proc" 2>/dev/null || true
+		fi
 
 		# some container environment are used at second-stage, it already treats /proc and so on
 		if [ -z "$(ls -A "$TARGET/proc")" ]; then
